src/settings/Settings.tsx

```diff
@@ -956,4 +956,8 @@ export const SETTINGS: {[setting: string]: ISetting} = {
         supportedLevels: LEVELS_UI_FEATURE,
         default: true,
     },
+    [UIFeature.ForceRoomExportParameters]: {
+        supportedLevels: LEVELS_UI_FEATURE,
+        default: {},
+    },
```

Comment:
https://github.com/vector-im/element-web/blob/develop/docs/config.md#ui-features

> Parts of the UI can be disabled using UI features. These are settings which appear under settingDefaults and can only be true (default) or false. When false, parts of the UI relating to that feature will be disabled regardless of the user's preferences.

UIFs must be booleans